### PR TITLE
(fix) minor refactor to include a setter method for AspectRoutingGmsC…

### PR DESCRIPTION
**Pull request by yangyangv2**
…lientManager

## Checklist

- [X] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [X] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

File: restli-resources/src/main/java/com/linkedin/metadata/restli/AspectRoutingGmsClientManager.java [hunk 1]
```diff
@@ -27,6 +29,7 @@ public RoutingAspectConfig(BaseAspectRoutingGmsClient gmsClient, String setterNa
 
   public <ASPECT extends RecordTemplate> void registerRoutingGmsClient(@Nonnull Class<ASPECT> routingAspectClass,
       @Nonnull String routingAspectSetterName, @Nonnull BaseAspectRoutingGmsClient routingGmsClient) {
+    log.info("Registering routing gms clients: {}, {}, {}", routingAspectClass, routingAspectSetterName, routingGmsClient);
```
Comment (zhixuan1120) [inline restli-resources/src/main/java/com/linkedin/metadata/restli/AspectRoutingGmsClientManager.java:32]: I'm not sure if {} just works for Class type. Maybe use routingAspectClass.getCanonicalName() instead ?

Comment (zhixuanjia) [inline restli-resources/src/main/java/com/linkedin/metadata/restli/AspectRoutingGmsClientManager.java:32]: I'm not sure {} just works for Class type, maybe use routingAspectClass.getCanonicalName() ?

Comment (yangyangv2) [reply restli-resources/src/main/java/com/linkedin/metadata/restli/AspectRoutingGmsClientManager.java]: it should be calling String.valueOf(), but I'll get the canonical name for better output.

**Review by zhixuanjia — APPROVED** [review]
LGTM

**Pull request merged**